subprojects/core/src/main/java/org/gradle/api/tasks/SourceTask.java

```diff
@@ -68,7 +69,7 @@ protected PatternFilterable getPatternSet() {
     @InputFiles
     @SkipWhenEmpty
     @IgnoreEmptyDirectories
-    @ToBeReplacedByLazyProperty
+    @NotToBeReplacedByLazyProperty(because = "SourceTask will be deprecated")
```

Comment:
❓ Should we also do the same for `VerificationTask.getIgnoreFailures()`?

Reply:
That's an interface that doesn't cause any problems with this validation, so I wouldn't do it as part of this PR. `SourceTask` I had to change because there's no other way to avoid the warnings.